openedx_external_enrollments/migrations/0003_auto_20210310_2228.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2021-03-10 22:28
```

Comment:
I think this migration can be avoided and configure the field again. @ivanvgh

Reply:
Removed migration and recreated inside 0002_other_course_settings.py.